QuickPay/Services/SubscriptionsService.cs

```diff
@@ -92,7 +92,6 @@ public Task<Subscription> CancelSubscription(int id, string callbackUrl = null,
             };
 
             var url = "subscriptions/" + id + "/cancel";
-            url += "?synchronized=" + synchronized;
```

Comment:
`?synchronized` do not care about the value, just it's presents.
So `?synchronized` should only be added if `synchronized == true`
ex.
```
if (synchronized) {
  url += "?synchronized";
}
  ```

Reply:
Hi Jens
Thanks for the info, I didn't know that about the synchronized query param.

But it hardly matters anyways since this PR is removing the synchronized part that is already there. This is instructed by QuickPay so I just follow orders 😄 

Would you please revisit your review

Reply:
oh Yeah! Sorry my bad :)

Reply:
Thanks. Have a nice weekend 🙂